Update to SL Custom Integration Section (#7623)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud-integrations/avail-sl-integrations.md

### Custom integration

> - All BI tools can use [exports](/docs/use-dbt-semantic-layer/exports) with the <Constant name="semantic_layer" />, even if they don’t have a native integration.

Author: ```suggestion
- all BI tools can use [exports](/docs/use-dbt-semantic-layer/exports) with the <Constant name="semantic_layer" />, even if they don’t have a native integration.
```

> - all BI tools can use [exports](/docs/use-dbt-semantic-layer/exports) with the <Constant name="semantic_layer" />, even if they don’t have a native integration.

Author: ```suggestion
- All BI tools can use [exports](/docs/use-dbt-semantic-layer/exports) with the <Constant name="semantic_layer" />, even if they don’t have a native integration.
```

> - [Exports](/docs/use-dbt-semantic-layer/exports) use exports with BI tools with the <Constant name="semantic_layer" />, even if you don’t have a native integration.

Reviewer: ```suggestion
- All BI tools can use [exports](/docs/use-dbt-semantic-layer/exports) with the <Constant name="semantic_layer" />, even if they don’t have a native integration.
```
